### PR TITLE
Practice-one

### PR DESCRIPTION
**Pull request by TIEN1909**
#134 practice-one exercise-one

### CODE REVIEW

File: javascript-training/practice-one/exercise-one/about-me/aboutme.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
```
Comment (hungdo-agilityio) [inline javascript-training/practice-one/exercise-one/about-me/aboutme.html:1]: Please help update naming file `aboutme` -> `about-me`

File: javascript-training/practice-one/exercise-one/about-me/aboutme.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+<body>
+    <h1>About Me</h1>
+
+    <ul>
+        <li>Nickname: <span id="nickname"></span> </li>
+        <li>Favorites: <span id="favorites"></span></li>
+        <li>Hometown: <span id="hometown"></span> </li>
+    </ul>
+
+    <div class="imageContainer"></div>
```
Comment (hungdo-agilityio) [inline javascript-training/practice-one/exercise-one/about-me/aboutme.html:18]: `imageContainer` -> `image-container`. Please help to recheck for all

File: javascript-training/practice-one/exercise-one/about-me/aboutme.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+if (favorite) {
+  favorite.innerHTML = "Play football";
+}
+
+if (homeTown) {
+  homeTown.innerHTML = "Kon Tum";
+}
+
+// Iterate through each li and change the class to "listitem".
+//Add a style tag that sets a rule for "listitem" to make the color red.
```
Comment (hungdo-agilityio) [inline javascript-training/practice-one/exercise-one/about-me/aboutme.js:24]: Add space after `//` Please recheck for the rest

File: javascript-training/practice-one/exercise-one/about-me/aboutme.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+listItems.forEach((listItem) => {
+  listItem.classList.add("list-item");
+  listItem.style.color = "red";
```
Comment (hungdo-agilityio) [inline javascript-training/practice-one/exercise-one/about-me/aboutme.js:27]: This is not the right one.  Please use class to styling

File: javascript-training/practice-one/exercise-one/about-me/aboutme.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+});
+
+//Create a new img element and set its src attribute to a picture of you. Append that element to the page.
+const img = document.createElement("img");
+if (img) {
```
Comment (hungdo-agilityio) [inline javascript-training/practice-one/exercise-one/about-me/aboutme.js:32]: Is any chance that img has falsy value?

Comment (hungdo-agilityio) [reply javascript-training/practice-one/exercise-one/about-me/aboutme.js:32]: Add breakline before and after each `if`. Please recheck for all

File: javascript-training/practice-one/exercise-one/the-book-list/app.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
+const books = [
+  {
+    title: "The Design of EveryDay Things",
+    url: "https://salt.tikicdn.com/cache/w1200/ts/product/bb/8e/21/8622d78f10b85cf520709d8ce29ac983.jpg",
+    author: "Don Norman",
+    alreadyRead: false,
```
Comment (hungdo-agilityio) [inline javascript-training/practice-one/exercise-one/the-book-list/app.js:6]: I recommend for props, variables that is boolean should have the prefix `is`, `has`. For example: `isRead`, `hasSomething`

File: javascript-training/practice-one/exercise-one/the-book-list/app.js [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+  {
+    title: "The Most Human Human",
+    url: "https://images.penguinrandomhouse.com/cover/9780307476708",
+    author: "Brian Christian",
+    alreadyRead: true,
+  },
+];
+
+const bookList = document.createElement("ul");
+books.forEach((book) => {
```
Comment (hungdo-agilityio) [inline javascript-training/practice-one/exercise-one/the-book-list/app.js:17]: Can we destruct `book` here?

File: javascript-training/practice-one/exercise-one/the-book-list/app.js [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+  bookImage.src = book.url;
+  bookItem.appendChild(bookImage);
+
+  const bookDescription = document.createTextNode(book.title + " by " + book.author);
+  bookP.appendChild(bookDescription);
+  bookItem.appendChild(bookP);
+
+  if (book.alreadyRead) {
+    bookItem.style.color = "red";
```
Comment (hungdo-agilityio) [inline javascript-training/practice-one/exercise-one/the-book-list/app.js:30]: Please use `class` for styling instead

File: javascript-training/practice-one/exercise-one/the-book-list/app.js [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+const bookList = document.createElement("ul");
+books.forEach((book) => {
+  const bookP = document.createElement("p");
+  const bookItem = document.createElement("li");
+  const bookImage = document.createElement("img");
+
+  bookImage.src = book.url;
+  bookItem.appendChild(bookImage);
+
+  const bookDescription = document.createTextNode(book.title + " by " + book.author);
```
Comment (hungdo-agilityio) [inline javascript-training/practice-one/exercise-one/the-book-list/app.js:25]: https://eslint.org/docs/latest/rules/prefer-template

File: javascript-training/practice-one/exercise-one/the-logo-hijack/app.js [hunk 10]
```diff
@@ -0,0 +1,13 @@
+const logoGoogle = document.querySelector("img.lnXdpd");
```
Comment (hungdo-agilityio) [inline javascript-training/practice-one/exercise-one/the-logo-hijack/app.js:1]: Please help to add comments to this file. I dont' know where this script used

File: javascript-training/practice-one/exercise-one/the-logo-hijack/app.js [hunk 11]
```diff
@@ -0,0 +1,16 @@
+// Select the Google logo image element
+const logoGoogle = document.querySelector("img.lnXdpd");
+
+// Select the Google search button element
+const btnSearch = document.querySelector("input.gNO89b");
```
Comment (tungtran-agilityio) [inline javascript-training/practice-one/exercise-one/the-logo-hijack/app.js:5]: ?

File: javascript-training/package.json [hunk 12]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (tungtran-agilityio) [inline javascript-training/package.json:1]: I think you use create npm project for each exercise

**Review by tungtran-agilityio — CHANGES_REQUESTED** [review]
Please init npm project for each exercise

**Review by tungtran-agilityio — APPROVED** [review]

**Pull request closed without merging**